{"level":"error","message":"Failed to sync database: connect ECONNREFUSED 127.0.0.1:3306","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-61,"fatal":true,"port":3306,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-61,"fatal":true,"port":3306,"syscall":"connect"},"stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306\n    at new BaseError (/home/project/node_modules/sequelize/lib/errors/base-error.js:13:5)\n    at new ConnectionError (/home/project/node_modules/sequelize/lib/errors/connection-error.js:35:5)\n    at new ConnectionRefusedError (/home/project/node_modules/sequelize/lib/errors/connection/connection-refused-error.js:30:5)\n    at ConnectionManager.connect (/home/project/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:92:17)\n    at async ConnectionManager._connect (/home/project/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)\n    at async eval (/home/project/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:174:32)\n    at async ConnectionManager.getConnection (/home/project/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:197:7)\n    at async eval (/home/project/node_modules/sequelize/lib/sequelize.js:305:26)","timestamp":"2025-04-28T18:49:51.389Z"}
{"level":"error","message":"Failed to sync database: connect ECONNREFUSED 127.0.0.1:3306","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-61,"fatal":true,"port":3306,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-61,"fatal":true,"port":3306,"syscall":"connect"},"stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306\n    at new BaseError (/home/project/node_modules/sequelize/lib/errors/base-error.js:13:5)\n    at new ConnectionError (/home/project/node_modules/sequelize/lib/errors/connection-error.js:35:5)\n    at new ConnectionRefusedError (/home/project/node_modules/sequelize/lib/errors/connection/connection-refused-error.js:30:5)\n    at ConnectionManager.connect (/home/project/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:92:17)\n    at async ConnectionManager._connect (/home/project/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)\n    at async eval (/home/project/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:174:32)\n    at async ConnectionManager.getConnection (/home/project/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:197:7)\n    at async eval (/home/project/node_modules/sequelize/lib/sequelize.js:305:26)","timestamp":"2025-04-28T18:52:08.926Z"}
{"level":"error","message":"Failed to sync database: Connection lost: The server closed the connection.","name":"SequelizeConnectionError","original":{"code":"PROTOCOL_CONNECTION_LOST","fatal":true},"parent":{"code":"PROTOCOL_CONNECTION_LOST","fatal":true},"stack":"SequelizeConnectionError: Connection lost: The server closed the connection.\n    at new BaseError (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/errors/base-error.js:13:5)\n    at new ConnectionError (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/errors/connection-error.js:35:5)\n    at ConnectionManager.connect (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:102:17)\n    at async ConnectionManager._connect (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)\n    at async eval (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:174:32)\n    at async ConnectionManager.getConnection (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:197:7)\n    at async eval (/home/projects/sb1-jwpyswux/node_modules/sequelize/lib/sequelize.js:305:26)","timestamp":"2025-04-28T19:30:10.675Z"}
